### PR TITLE
Group related lines together by printed headers

### PR DESCRIPTION
**Pull request by croaky**
* "Installing Homebrew packages ..."
* "Restarting services ..."
* "Relinking OpenSSL ..."
* "Configuring Ruby ..."
* "Running your customizations from ~/.laptop.local ..."
* "Cleaning up old Homebrew formulas ..."

To make that work in the desired way,
print less noise in the custom functions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,28 +99,32 @@ It should take less than 15 minutes to install (depends on your machine).
 Customize in `~/.laptop.local`
 ------------------------------
 
-Your `~/.laptop.local` is run at the end of the Laptop script.
+Your `~/.laptop.local` is run near the end of the Laptop script.
```
Comment (gabebw) [inline README.md]: It looks like the very last thing we do in the `mac` script  is run `~/.laptop.local`. What am I missing?

Comment (croaky) [inline README.md]: I should have done this in https://github.com/thoughtbot/laptop/pull/432, just made it part of that change.

File: mac [hunk 2]
```diff
@@ -138,40 +133,43 @@ if brew list | grep -Fq brew-cask; then
...
-# shellcheck disable=SC2016
-append_to_zshrc 'eval "$(rbenv init - --no-rehash)"' 1
+fancy_echo "Restarting services ..."
+brew_launchctl_restart 'postgresql'
+brew_launchctl_restart 'redis'
 
+fancy_echo "Relinking OpenSSL ..."
 brew_install_or_upgrade 'openssl'
 brew unlink openssl && brew link openssl --force
-brew_install_or_upgrade 'libyaml'
```
Comment (gabebw) [inline mac]: I think this is related to SSL, and should stay here, though I'm not sure how it's related.

Comment (croaky) [inline mac]: I think it historically has had to go after the openssl change and before the Ruby installation.

**Comment by croaky** [comment]
I like the printed headers but not sure "Homebrew" is descriptive enough. Could take the comment idea from https://github.com/thoughtbot/laptop/pull/392/files and use the printed header idea here.

**Comment by croaky** [comment]
Made the middle section more specific:

```
Updating Homebrew formulae ...
Already up-to-date.

Updating Unix tools ...
Warning: thoughtbot/formulae/rcm-1.3.0 already installed

Updating Heroku tools ...
Warning: thoughtbot/formulae/parity-0.9.2.1 already installed

Updating GitHub tools ...

Updating image tools ...

Updating testing tools ...

Updating programming languages ...

Updating databases ...
/Users/croaky/Library/LaunchAgents/homebrew.mxcl.postgresql.plist -> /usr/local/opt/postgresql/homebrew.mxcl.postgresql.plist
/Users/croaky/Library/LaunchAgents/homebrew.mxcl.redis.plist -> /usr/local/opt/redis/homebrew.mxcl.redis.plist

Updating OpenSSL ...
Unlinking /usr/local/Cellar/openssl/1.0.2e... 1556 symlinks removed
Linking /usr/local/Cellar/openssl/1.0.2e... 1556 symlinks created
```

File: mac [hunk 3]
```diff
@@ -141,41 +132,51 @@ fancy_echo "Updating Homebrew formulae ..."
...
+fancy_echo "Updating image tools ..."
+brew_install_or_upgrade 'imagemagick'
 
-brew_install_or_upgrade 'openssl'
-brew unlink openssl && brew link openssl --force
+fancy_echo "Updating testing tools ..."
+brew_install_or_upgrade 'qt'
+
+fancy_echo "Updating programming languages ..."
 brew_install_or_upgrade 'libyaml'
```
Comment (gabebw) [inline mac]: Should we add a comment on this line that it should happen after `openssl`?

**Comment by gabebw** [comment]
Cool, looks good.

**Pull request merged**